modules/ui/app/scripts/controllers/DialogModalCtrl.js

```diff
@@ -2,12 +2,12 @@
 
 angular.module('wasabi.controllers')
     .controller('DialogModalCtrl',
-        ['$scope', '$modalInstance', 'options', '$timeout',
-            function ($scope, $modalInstance, options, $timeout) {
+        ['$scope', '$modalInstance', 'options', '$timeout', '$sce',
```

Comment:
That's a built-in variable from Angular JS (1.x).  I had to look up what it stands for.  😄 I don't believe they have implemented it as a longer named variable, but I'll add a comment, "Strict Contextual Escaping".